geode-core/src/main/java/org/apache/geode/internal/net/BufferPool.java

```diff
@@ -253,7 +253,7 @@ ByteBuffer expandWriteBufferIfNeeded(BufferType type, ByteBuffer existing,
     return newBuffer;
   }
 
-  ByteBuffer acquireDirectBuffer(BufferPool.BufferType type, int capacity) {
+  public ByteBuffer acquireDirectBuffer(BufferPool.BufferType type, int capacity) {
```

Comment:
what required the addition of `public`?

Reply:
it's now accessed by the org.apache.geode.internal.tcp package